Keas.Mvc/Services/DocumentSigningService.cs

```diff
@@ -30,19 +33,33 @@ public class DocumentSigningService : IDocumentSigningService
 
         private readonly DocumentSigningSettings _documentSigningSettings;
 
+        private OAuth.UserInfo _userInfo;
+
         public DocumentSigningService(IOptions<DocumentSigningSettings> documentSigningSettings)
         {
-            _apiClient = _apiClient ?? new ApiClient();
+            _apiClient ??= new ApiClient();
```

Comment:
eww

Reply:
Yeah, I hope that isn't the new and shiny. I just got used to ?? :)

Reply:
@srkirkland What docusign create page are you referring to?

Reply:
The team admin one. It used to have some basic instructions but there's nothing on the new one.

Reply:
@sprucely I think @srkirkland means the ones here:
https://computing.caes.ucdavis.edu/documentation/peaks/docusign-documents

Because you updated the page in PEAKS I believe.